src/defs.h

```diff
@@ -174,15 +210,21 @@ typedef struct {
 } regfile_t;
 
 /* phase-2 IR definition */
-typedef struct {
+struct ph2_ir_t {
     opcode_t op;
     int src0;
     int src1;
     int dest;
     char func_name[MAX_VAR_LEN];
     char true_label[MAX_VAR_LEN];
     char false_label[MAX_VAR_LEN];
-} ph2_ir_t;
+    struct basic_block_t *next_bb;
```

Comment:
Enforce use of `typedef`. That is, prefer `basic_block_t *next_bb` over `struct basic_block *bb`.